Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorLevelTracingEffect.py

```diff
@@ -163,11 +163,14 @@ def preview(self,xy):
     ijk = self.effect.xyToIjk(xy, self.sliceWidget, masterImageData, parentTransformNode)
     dimensions = masterImageData.GetDimensions()
 
-    for index in range(3):
+    # LevelTracing implementation is 2D, the next set of lines should be uncommented when
+    # 3D LevelTracing is implemented 
+    
+    # for index in range(3):
       # TracingFilter crashes if it receives a seed point at the edge of the image,
       # so only accept the point if it is inside the image and is at least one pixel away from the edge
-      if ijk[index] < 1 or ijk[index] >= dimensions[index]-1:
-        return
+      # if ijk[index] < 1 or ijk[index] >= dimensions[index]-1:
+        # return
```

Comment:
I guess you confirmed it doesn't crash in this case?  Can you update the comment and remove the dead code?

Reply:
The check can only be removed along the slice normal axis, but the check must remain along the two other axes. Unless you find evidence that the crash is fixed (there is a relevant code change and your own tests confirm that you can click on boundary pixels and there is no crash).

Reply:
This is not built and tested, I might have misunderstood @lassoan in the discourse forum. I thought commenting these lines would be the solution and everyone will be happy

Reply:
I will build and test now

Reply:
Thanks for working on this @sedghi.

The test is needed in the slice plane (unless you prove otherwise), but not outside the slice plane.

No need to build, you can just change the .py file and restart Slicer.

Reply:
I've checked what I wrote in my commend on discourse. I meant that you can simply remove the check *on your own computer* - because you know that you must not click at boundary voxels. But in Slicer core this workaround is not good enough because users will not know that they will make the application crash by clicking at boundary voxels.